src/Bundle/Resources/config/services/integrations/doctrine/phpcr-odm.xml

```diff
@@ -16,9 +16,9 @@
         <defaults public="true" />
 
         <service id="sylius.grid_driver.doctrine.phpcrodm" class="Sylius\Bundle\GridBundle\Doctrine\PHPCRODM\Driver">
-            <argument type="service" id="doctrine_phpcr.odm.document_manager" />
+            <argument type="service" id="doctrine_phpcr.odm.document_manager" on-invalid="ignore" />
```

Comment:
The package could be available but the service could me missing if odm is not configured in Doctrine bundle.